pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -707,14 +719,20 @@ func (ctrl *Controller) syncImageConfig(key string) error {
 }
 
 func registriesConfigIgnition(templateDir string, controllerConfig *mcfgv1.ControllerConfig, role string,
```

Comment:
(Non-blocking: The return value from this function no longer contains only the `registries.conf` data; maybe it should be renamed. (OTOH, arguably, the name refers to “registries-related configuration”.))

---

(It’s also not quite ideal that `RunImageBootstrap` invokes this with always-empty `*Regs` parameters, but splitting this into two functions and then somehow merging the ignition configuration in `syncImageConfig` would probably not be cleaner.)

Reply:
Yeah, I don't want to mess around with this now since we need this in 4.2 and feature freeze is already over. Will try to do something for it in 4.3.